ctru-rs/src/services/cam.rs

```diff
@@ -0,0 +1,1050 @@
+use bitflags::bitflags;
+use ctru_sys::Handle;
+use crate::services::gspgpu::FramebufferFormat;
+
+pub struct Cam(());
```

Comment:
It would be great to have some documentation on the structs/enums and functions (and a quick line for the module to explain what it is for).